api/src/test/java/org/openmrs/module/appframework/service/AppFrameworkServiceTest.java

```diff
@@ -426,6 +438,33 @@ public void purgeUserApp_shouldRemoveTheUserAppFromTheDatabaseAndUpdateTheListOf
 		assertEquals(--originalAppDescriptorCount, appFrameworkService.getAllApps().size());
 	}
 	
+	@DirtiesContext
+	@Test
+	public void getLoginLocations_() {
```

Comment:
Can we change the name to what tells what this test is about? 😊

Reply:
Good catch! It was an oversight